cluster-scope/overlays/nerc-ocp-prod/feature/external-console-access/api-proxy/role.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Didn't know what this role was doing, and after some looking around I landed here: 
https://docs.openshift.com/container-platform/4.10/authentication/managing-security-context-constraints.html#role-based-access-to-ssc_configuring-internal-oauth

Question: Can non-admins create a role like this in their namespace and run their containers as root?

Reply:
@naved001 if someone tries to create a rolebinding that grants permissions they don't already have, it will fail with:

```
Error from server (Forbidden): error when creating ".": roles.rbac.authorization.k8s.io "allow-anyuid" is forbidden: user "testuser" (groups=["system:authenticated:oauth" "system:authenticated"]) is attempting to grant RBAC permissions not currently held:
{APIGroups:["security.openshift.io"], Resources:["securitycontextconstraints"], ResourceNames:["anyuid"], Verbs:["use"]}
Error from server (NotFound): error when creating ".": roles.rbac.authorization.k8s.io "allow-anyuid" not found
```